packages/tupaia-web-server/src/routes/DashboardsRoute.ts

```diff
@@ -17,9 +17,10 @@ export class DashboardsRoute extends Route<DashboardsRequest> {
     const project = (
       await ctx.services.central.fetchResources('projects', {
         filter: { code: projectCode },
-        columns: JSON.stringify(['entity.code', 'entity_hierarchy.name']),
```

Comment:
this is temporary code that Ethan is refactoring in another PR so I wouldn't be too worried about it